geoips/plugins/modules/readers/seviri_hrit.py

```diff
@@ -59,43 +65,155 @@
 
 IR_CALIB = {
     "msg1": {
-        "B04": {"wn": 2567.330, "a": 0.9956, "b": 3.410},  # IR3.9 Water vapour channel
-        "B05": {"wn": 1598.103, "a": 0.9962, "b": 2.218},  # IR6.2 Water vapour channel
-        "B06": {"wn": 1362.081, "a": 0.9991, "b": 0.478},  # IR7.3 Water vapour channel
-        "B07": {"wn": 1149.069, "a": 0.9996, "b": 0.179},  # IR8.7 Atmospheric window
-        "B08": {"wn": 1034.343, "a": 0.9999, "b": 0.060},  # IR9.7 Ozone channel
-        "B09": {"wn": 930.647, "a": 0.9983, "b": 0.625},  # IR10.8 Atmospheric window
-        "B10": {"wn": 839.660, "a": 0.9988, "b": 0.397},  # IR12.0 Atmospheric window
+        "B04": {
```

Comment:
The previous code was much more readable - can we revert to it?

Reply:
reverted back, relevant ([commit ID](https://github.com/NRLMMD-GEOIPS/geoips/commit/6b1a95b701cb6b5d74b9200994ca34742d33e706)). Thanks for catching this, @biosafetylvl5